daemons/dss-index/app.py

```diff
@@ -33,16 +33,18 @@ def dispatch_s3_indexer_event(event, context) -> None:
     if event.get("Event") == "s3:TestEvent":
         logger.info("DSS index daemon received S3 test event")
     else:
-        _handle_event(Replica.aws, event, context)
+        for event_record in event["Records"]:
```

Comment:
Why was this change needed?

Reply:
will indexer receive multiple records to process in one request?

Reply:
SQS->Lambda sends messages in batches:
https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html

Reply:
From https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html
> All messages in a failed batch return to the queue, so your function code must be able to process the same message multiple times without side effects.

Will this be a problem?

Reply:
Not as long as the indexer is idempotent, which I believe it is.

Also, if needed, batch size may be configured.

Reply:
adding idempotent test for indexer #1697

Reply:
Fixes the failing tests and i'll review.